clients/include/testing_ormtr_unmtr.hpp

```diff
@@ -623,9 +623,8 @@ void testing_ormtr_unmtr(Arguments& argus)
 
     // validate results for rocsolver-test
     // using n * machine_precision as tolerance
-    int s = left ? m : n;
     if(argus.unit_check)
-        ROCSOLVER_TEST_CHECK(T, max_error, s);
+        ROCSOLVER_TEST_CHECK(T, max_error, (left ? m : n));
```

Comment:
Just noticed that `n * machine_precision` should probably have been `s * machine_precision`. Also, same comment as above.